Presentation.Web/Hangfire/KeepReadModelsInSyncProcess.cs

```diff
@@ -30,17 +31,28 @@ public void Execute(BackgroundProcessContext context)
             CoolDown();
         }
 
-        private static void ProcessItSystemUsageModule(IBackgroundJobLauncher backgroundJobLauncher,
-            CancellationTokenSource combinedTokenSource)
+        private static void ProcessItSystemUsageModule(IBackgroundJobLauncher backgroundJobLauncher, CancellationTokenSource combinedTokenSource)
         {
+
```

Comment:
køres før (vigtigt) og efter (for en sikkerhedsskyld hvis vi kommer til at lave ged i den senere)